android/PartySampleNetworkCommon/lib/NetworkManager.cpp

```diff
@@ -169,10 +169,10 @@ NetworkManager::CreateAndConnectToNetwork(
     PartyNetworkConfiguration cfg = {};
 
     // Setup the network to allow the maximum number of single-device players of any device type
-    cfg.maxDeviceCount = c_maxNetworkConfigurationMaxDeviceCount;
+    cfg.maxDeviceCount = 30;
     cfg.maxDevicesPerUserCount = 1;
     cfg.maxEndpointsPerDeviceCount = 1;
-    cfg.maxUserCount = c_maxNetworkConfigurationMaxDeviceCount;
+    cfg.maxUserCount = 1;
```

Comment:
this doesn't look correct. why would you only want one unique user in the network across all devices?

Reply:
this comment applies to your other edits as well